src/SaveHandler/MongoDB.php

```diff
@@ -10,10 +10,10 @@
 
 namespace Zend\Session\SaveHandler;
 
-use Mongo;
-use MongoClient;
-use MongoCollection;
-use MongoDate;
+use MongoDB\BSON\Binary;
+use MongoDB\BSON\UTCDatetime;
+use MongoDB\Client as MongoClient;
```

Comment:
We can, but it requires far more code and code complexity. The Mongo PHP Library exists to provide the convenience layer that was previously in the extension, but in a way that can be more quickly and easily updated as new features are added and/or bugfixes released. Considering most developers will now be using the extension / PHP library combination if using MongoDB at all on PHP, this is not an unreasonable requirement.